AUTHORS.rst

```diff
@@ -4,6 +4,10 @@ Authors
 The DAGMC project began as the CAD-MCNPX project in 2003, and since then has
 included contributions from the following:
 
+* Paul Romano
+* Jonathan Shimwell
+* Chelsea D'Angelo
+* Xiaokang Zhang
```

Comment:
It seems like the rest of the names are in alphabetical order. I feel we should continue that practice